src/web/core/src/test/java/org/geoserver/web/data/resource/ResourceConfigurationPageTest.java

```diff
@@ -926,7 +927,7 @@ public void testCustomizeAttributes() {
         form.setValue(formEdit1 + "2:component:text", "abstract");
         String cql = "Concatenate(description, ' and more!')";
         form.setValue(formEdit1 + "4:component:area", cql);
```

Comment:
Could you customize the description and check the update worked below, in "check saving happened"?

Reply:
done